.gitmodules

```diff
@@ -1,9 +1,12 @@
 [submodule "apm-protocol/apm-network/src/main/proto"]
 	path = apm-protocol/apm-network/src/main/proto
-	url = https://github.com/apache/skywalking-data-collect-protocol.git
+	url = file:///data/developer/git/java/skywalking/apm-protocol/apm-network/src/main/proto
+	#url = https://github.com/apache/skywalking-data-collect-protocol.git
 [submodule "oap-server/server-query-plugin/query-graphql-plugin/src/main/resources/query-protocol"]
 	path = oap-server/server-query-plugin/query-graphql-plugin/src/main/resources/query-protocol
-	url = https://github.com/apache/skywalking-query-protocol.git
+	url = file:///data/developer/git/java/skywalking/oap-server/server-query-plugin/query-graphql-plugin/src/main/resources/query-protocol
+	#url = https://github.com/apache/skywalking-query-protocol.git
 [submodule "skywalking-ui"]
 	path = skywalking-ui
-	url = https://github.com/apache/skywalking-rocketbot-ui.git
+	#url = https://github.com/apache/skywalking-rocketbot-ui.git
+	url = file:///data/developer/git/java/skywalking/skywalking-ui
```

Comment:
This is unexpected changes

Reply:
> This is unexpected changes

Yes, for local testing, will revert and recheck, once done, I'll ping you, thanks :)